src/main/kotlin/com/exactpro/th2/pico/operator/config/ApplicationConfig.kt

```diff
@@ -28,5 +28,6 @@ data class ApplicationConfig(
     val rabbitMQManagement: RabbitMQManagementConfig,
     val grpc: GrpcConfig,
     val prometheus: PrometheusPortsConfig,
-    val defaultSchemaConfigs: DefaultSchemaConfigs
+    val defaultSchemaConfigs: DefaultSchemaConfigs,
+    val defaultEnvironmentVariables: Map<String, Set<String>>
```

Comment:
We have added this to the configuration but who is using this parameter? I use that we assign it to some property but that is it... Maybe something is missing here?

Reply:
Looks strange, but this config is used in th2-pico

Reply:
Maybe then we should not add this here?

Reply:
do you propose add a separate config file or remove singleton configuration and extend config in pico project?

Reply:
I just propose not to add something that this project doesn't need...